charts/drax/charts/dashboard/templates/configmap-core-ui.yaml

```
@@ -14,5 +14,5 @@ name: {{ include "accelleran.common.fullname" (dict "top" $) }}-core-ui
 data:
   config.js: |-
     window.GRAFANA_PORT = "{{ tpl $.Values.config.grafanaPort $ }}";
-    window.WEBSOCKET_PORT = "{{ tpl $.Values.config.websocketPort $ }}";
+    window.WEBSOCKET_PORT = "{{ .Values.service.ports.websocket.nodePort }}";
```

To be safe in the templates maybe use `$.Values`, although I think it should be fine already as is in almost all templates.

```suggestion
    window.WEBSOCKET_PORT = "{{ $.Values.service.ports.websocket.nodePort }}";
```